### PR TITLE
LPS-163815 Test Automation Creation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-163815

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Tooltip.path [hunk 1]
```diff
@@ -35,6 +35,11 @@
 	<td>//div[contains(@class, 'popover-body')]//p[contains(@class, 'text-neutral') and contains(text(), '${key_content}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>POPOVER_HEADER</td>
+	<td>//div[contains(@class,'popover-header')][normalize-space(text())='${key_content}']</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Tooltip.path:40]: key_header

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! PR Updated. As it was a small change I will forward it. Thanks!

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**